UIAutomationTests/TestFixture.cs

```diff
@@ -0,0 +1,218 @@
+﻿using FlaUI.Core;
+using FlaUI.Core.AutomationElements;
+using FlaUI.Core.Conditions;
+using FlaUI.Core.Input;
+using FlaUI.Core.WindowsAPI;
+using FlaUI.UIA3;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UIAutomationTests
+{
+    public class TestFixture : IDisposable
+    {
+
+        string vsExecutablePath = $@"C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\Common7\IDE\devenv.exe";
+        string slnPath = $@"C:\repos\powerbi-exasol\Exasol\Exasol.mproj";
+
+        string queryPqPath = $@"C:\repos\powerbi-exasol\Exasol\Exasol.query.pq";
+        string originalQueryPqStr;
+
+        Application app;
+        UIA3Automation automation;
+
+        ConditionFactory cf;
+
+        AutomationElement mainWindow;
+
+        AutomationElement debugTargetButtonAE;
+        Button debugTargetButton;
+        AutomationElement MQueryOutput;
+        AutomationElement[] tabItemAEs;
+
+        //https://stackoverflow.com/questions/12976319/xunit-net-global-setup-teardown
+        //Do "global" initialization here; Only called once.
+        public TestFixture()
+        {
+            PrepVisualStudio();
+
+            SetupConditionFactory();
+
+            AcquireDebugTargetButton();
+            PressDebugTargetButton();
+
+            AcquireMQueryWindowAndAcquireTabsWhenFullyLoaded();
+
+            //entering the credentials seems to be more reliable
+            EnterCredentials();
+
+            SaveAndBackupOriginalQueryString();
+        }
+
+        private void SetupConditionFactory()
+        {
+            cf = new ConditionFactory(new UIA3PropertyLibrary());
+        }
+
+        private void EnterCredentials()
+        {
+            //the errors tab will pop up and ask for credentials
+            var errorsTabAE = tabItemAEs[2];
+            errorsTabAE.AsTabItem().Select();
+
```

Comment:
Small suggestion. Whenever you use space lines it's usually because you want to separate sections in a method.
A better way to achieve that and make it more readable is to use the same level of abstraction in your method, and extracting these small sections to their own dedicated method with a meaningful name, derived by its abstraction, not its implementation (ie, a name useful from the POV of the user of the method).

Reply:
I could split this up into smaller functions but this is basically 1 bigger action and since it's 10 LoC I didn't really see the need.
It's always acquiring an element, then action and repeat. I'll leave it like this for now if that's OK with you but it's a good suggestion.